src/components/Layout/AsideMenu/MenuItems.jsx

```diff
@@ -0,0 +1,78 @@
+import { MenuList, Icon, MenuItem, Box, Text } from '@chakra-ui/react';
+import React from 'react';
+import { AiOutlineTeam } from 'react-icons/ai';
+import { MdOutlinePending } from 'react-icons/md';
+import { RiQuestionnaireLine, RiInformationLine, RiProfileLine } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+
+function MenuWrap({ menuType, children }) {
```

Comment:
1. We should extract `MenuWrap` and `MenuLink` to separate files (one component per file)
2. A good practice would be to create an enum for menu type like so:

```js
// in file constants/menu-type.js
export default const MENU_TYPES = {
    MOBILE: 'mobile',
    DESKTOP: 'desktop',
}
```

and then use that for comparison

```js
if (menuType === MENU_TYPE.MOBILE) { ....
```